react-native-ChartView.js

```diff
@@ -154,6 +153,34 @@ class ChartWeb extends Component {
     };
 };
 
+var flattenObject = function (obj, str='{') {
```

Comment:
@Noiwex Yes, didn't mean to include both changes in this PR. Thanks for catching it

@jonrh created it myself to solve that problem.